string comand
float32 value
---